### PR TITLE
chore: update init script for 0.7.0 release

### PR DESCRIPTION
**Pull request by lino-levan**
Meant to be merged along with #118

### CODE REVIEW

**Comment by lino-levan** [comment]
@hashrock PTAL!

File: init.ts [hunk 1]
```diff
@@ -64,14 +64,8 @@ const DENO_JSONC_CONTENTS = `{
     "dev": "deno run --allow-net --allow-read --allow-env --watch main.tsx --dev",
     "serve": "deno run --allow-net --allow-read --allow-env --no-check main.tsx",
   },
-  "importMap": "./import_map.json"
```
Comment (hashrock) [inline init.ts:67]: 🙌

**Review by hashrock — APPROVED** [review]
LGTM!

Note(probably for me):
- Do not merge this immediately. You will need to merge the associated PRs and publish 0.6.0.

**Comment by lino-levan** [comment]
hold on, going to update this commit to bump deps

**Comment by lino-levan** [comment]
Should be good now. Would be nice to merge #106 along with this.

**Comment by lino-levan** [comment]
Hold on, maybe we want to merge the imports into the `deno.jsonc`? I'll change this PR to do that.

**Comment by lino-levan** [comment]
Let me investigate the test failure, probably just due to a flaky test.

**Review by hashrock — APPROVED** [review]
LGTM!

**Pull request merged**